lib/src/components/NxCheckbox/NxCheckbox.scss

```diff
@@ -18,3 +18,16 @@
     vertical-align: 4px;
   }
 }
+
+.nx-radio-checkbox__input:checked:disabled {
+  + .nx-radio-checkbox__control {
+    &.nx-checkbox__box {
+      background-color:var(--nx-color-interactive-background-disabled);
+      border-color: var(--nx-color-interactive-border-disabled);
+
+      .fa-check {
+        color: var(--nx-color-text-disabled);
+      }
+    }
+  }
+}
```

Comment:
Newline is missing again

Reply:
> Newline is missing again

Apologies. It is fixed now. I have configured the "insert final newline" option on VS code too. Not sure why it didn't add it at the end.
1d21fc54210e59da3b3c4d81f821950d8be89d95